tests/integration/helpers.py

```diff
@@ -355,3 +355,42 @@ async def generate_log_file(
         logger.error(e.stdout.decode())
         raise e
     return res.stdout
+
+
+async def get_pebble_plan(
```

Comment:
I'm doing it in an itest since I want to check the pebble plan is still [there after a `kubectl delete pod`](https://github.com/canonical/loki-k8s-operator/issues/358)